docs/asf-docs/Introduction_hyp3_mintpy.ipynb

```diff
@@ -0,0 +1,527 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction to time-series analysis of HyP3 product with MintPy"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "1. Introduction\n",
+    "This notebook shows you how to do time-series analysis against HyP3 product with MintPy. This tutorial assumes that you have already successfully installed the MintPy and created the conda environment.\n",
+    "Detail installation info is refferered at https://github.com/insarlab/MintPy."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "2. Run the application"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Before running the program, some package need to be installed and imported."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!conda install -y -c conda-forge hyp3_sdk"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os, sys\n",
+    "import re\n",
+    "import glob\n",
+    "import zipfile\n",
+    "import json\n",
+    "from typing import no_type_check\n",
+    "from datetime import datetime, timezone\n",
+    "import numpy as np\n",
+    "from argparse import ArgumentParser\n",
+    "import hyp3_sdk as sdk\n",
+    "from hyp3_sdk import asf_search"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "2.1. Download the hyp3 data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Set the home directory, project name, output directory, and granule_list file. The granule_list file includes the RTC file names that users wants to use them to produce the interferograms for the time-series analysis purpose. Users\n",
+    "usually provide this file preapred with ASF vertex (https://serach.asf.alaska.edu)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hyp3_mintpy_home = \"/home/jzhu4/projects/work/hyp3/hyp3-mintpy\"\n",
+    "projectname = \"case5\"\n",
+    "outdir = \"/media/jzhu4/data/hyp3-mintpy/case5\"\n",
+    "granule_list = \"/media/jzhu4/data/hyp3-mintpy/case5/granule.dat\"\n",
```

Comment:
Have you found a set of interferograms that create a compelling time series? I'd recommend hard-coding an example list of pairs, rather than asking the user to go through the trouble of generating one themselves.

Reply:
changed